pkg/plugin/plugin.go

```diff
@@ -61,11 +99,17 @@ type NotificationRequest struct {
 }
 
 type Plugin interface {
+	// GetInfo returns the corresponding plugin *Info, panics on failure
 	GetInfo() *Info
```

Comment:
> panics on failure

I wouldn't specify that here as this function should never fail. If it does, it's a bug but stating it here like this makes it sound like this was expected behavior (if it was, `GetInfo()` should also return an `error`).